### PR TITLE
Gets export report functions working

### PR DESCRIPTION
**Pull request by fullstackcaveman**
## Description

Got export report function working and tentatively put it in the Saved Cases component table.

Function is basic at the moment and only includes a limited amount of case data without any visualizations.
Additional direction from stakeholders required to produce a meaningful report.

Created new column in Saved Cases Table to hold download button in SavedCases.js.
Changed Download Link to a Download Button in PDFExportButton.js to save space in table.
Added variables to convert UTC case_date received from database to a locally formatted date in ExportGenerator.jsx.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by juancaruizc — APPROVED** [review]
Awesome job refactoring!

File: src/components/pages/SavedCases/SavedCases.js [hunk 1]
```diff
@@ -39,6 +41,28 @@ function SavedCases({ savedCases, setSavedCases, deleteBookmark }) {
...
+            {
+              case_id: record.case_number,
+              case_date: record.case_date,
+              judge_name: record.last_name,
+              case_origin: record.case_origin_city,
+              case_outcome: record.case_outcome,
+              country_of_origin: record.country_of_origin,
+              gender: record.gender,
+              type_of_violence: record.type_of_violence,
+            },
```
Comment (jglez) [inline src/components/pages/SavedCases/SavedCases.js:60]: Perhaps we can include the state in the Case Origin field?

We can use string interpolation to accomplish this on the `case_origin` property:
```
case_origin: `${record.case_origin_city}, ${record.case_origin_state}`
```

File: src/components/pages/SavedCases/SavedCases.js [hunk 2]
```diff
@@ -29,7 +31,7 @@ function SavedCases({ savedCases, setSavedCases, deleteBookmark }) {
       width: '5%',
     },
     {
-      title: 'PDF',
+      title: 'Download Case',
```
Comment (jglez) [inline src/components/pages/SavedCases/SavedCases.js:34]: How do we differentiate to the user what the difference is between "Download Case" and "Download Report"?

File: src/components/pages/SavedCases/SavedCases.js [hunk 3]
```diff
@@ -39,6 +41,28 @@ function SavedCases({ savedCases, setSavedCases, deleteBookmark }) {
       ),
       width: '5%',
     },
+    {
+      title: 'Download Report',
+      render: (text, record) => (
+        <PDFExportButton
+          key="export"
+          fileName={`Judge_${record.last_name}_Case_${record.case_number}`}
```
Comment (jglez) [inline src/components/pages/SavedCases/SavedCases.js:49]: Excellent file name convention

File: src/components/pages/Cases/PDFOverviewExport/ExportGenerator.jsx [hunk 4]
```diff
@@ -59,17 +59,22 @@ const styles = StyleSheet.create({
...
     <Document>
       <Page style={styles.page}>
         <Image source={hrfLogo} style={styles.logo} />
         <View style={styles.table}>
           <View style={styles.tableRow}>
             <View style={styles.tableCol}>
-              <Text style={styles.columnNameCell}>Case ID</Text>
+              <Text style={styles.columnNameCell}>Case #</Text>
             </View>
             <View style={styles.tableCol}>
```
Comment (jglez) [inline src/components/pages/Cases/PDFOverviewExport/ExportGenerator.jsx:76]: When exporting the PDF to the user, perhaps we can change the "Nation of Origin" field to "Country of Origin" so the language is consistent with the table on Saved Cases

i.e. `    <Text style={styles.columnNameCell}>Country of Origin</Text>`

**Review by jglez — APPROVED** [review]
Nice work, Chris!

File: src/components/pages/SavedCases/SavedCases.js (hunk 1 above)
Comment (fullstackcaveman) [reply src/components/pages/SavedCases/SavedCases.js:60]: Good idea. 
Once we get a handle on what should actually be in the reports, we can implement items like this.
UI/UX should be involved here as well to get a meaningful report generated.

File: src/components/pages/SavedCases/SavedCases.js (hunk 2 above)
Comment (fullstackcaveman) [reply src/components/pages/SavedCases/SavedCases.js:34]: I agree.
UI/UX input needed.

File: src/components/pages/SavedCases/SavedCases.js (hunk 3 above)
Comment (fullstackcaveman) [reply src/components/pages/SavedCases/SavedCases.js:49]: Thanks.
I wanted to make sure that the filename was both unique and easily to determine who/what.

File: src/components/pages/Cases/PDFOverviewExport/ExportGenerator.jsx (hunk 4 above)
Comment (fullstackcaveman) [reply src/components/pages/Cases/PDFOverviewExport/ExportGenerator.jsx:76]: I agree.

File: src/components/pages/SavedCases/SavedCases.js (hunk 2 above)
Comment (nicholasbus) [inline src/components/pages/SavedCases/SavedCases.js:34]: Like this clarification. The PDF title alone was a bit confusing. 👍

File: src/components/pages/Cases/PDFOverviewExport/PDFExportButton.js [hunk 5]
```diff
@@ -2,16 +2,17 @@ import React from 'react';
 import { PDFDownloadLink } from '@react-pdf/renderer';
 import ExportGenerator from './ExportGenerator.jsx';
 import { LoadingOutlined } from '@ant-design/icons';
+import { DownloadOutlined } from '@ant-design/icons';
```
Comment (nicholasbus) [inline src/components/pages/Cases/PDFOverviewExport/PDFExportButton.js:5]: Nice download indication

File: src/components/pages/Cases/PDFOverviewExport/ExportGenerator.jsx [hunk 6]
```diff
@@ -59,17 +59,22 @@ const styles = StyleSheet.create({
 });
 
 export function MyDoc(props) {
+  // case_date has a UTC format from the database
+  // convertDate/date formats it to a correctly configured date string
```
Comment (nicholasbus) [inline src/components/pages/Cases/PDFOverviewExport/ExportGenerator.jsx:63]: Appreciate the thought process / explanation comments!

**Review by nicholasbus — APPROVED** [review]
Everything looks good! Appreciate the explanation comments, thoughtful variable names, and useful naming in the table.

**Pull request merged**